app/main.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+app = Flask(__name__)
```

Comment:
Nitpick, I think two lines after imports for [PEP8](https://www.python.org/dev/peps/pep-0008/)?